partner_affiliate/views/res_partner_view.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Show always parent_id plus add affiliates -->
+    <record id="view_partner_form_add_affiliate" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field eval="1" name="priority"/>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="/form/sheet//div[hasclass('o_row')]/field[@name='parent_id']" position="attributes">
```

Comment:
@tarteo here directly use `field`

Reply:
An xpath is needed because there are multiple `parent_id`, but the selector should be used in a wiser way. Something like:
```
expr="//field[@name='child_ids']/form//field[@name='parent_id']"
```

Reply:
@pedrobaeza Noted!